### PR TITLE
Hansol/update api 5

### PR DESCRIPTION
**Pull request by hh1834**
Finished wiring the update api 

call api '/shopcarts/<int:customer_id>/<int:product_id>' with requested quantity in body. I checked that the quantity for an item with customer id and product id is updated to that quantity. 

If negative quantity is given, 400 is returned with a descriptive message

### CODE REVIEW

File: service/service.py [hunk 1]
```diff
@@ -133,24 +133,36 @@ def create_cart_item():
...
+    app.logger.info('Request to update shopcart item with customer_id: %s, product_id: %s', customer_id, product_id)
+    cart_item = Shopcart.find_by_customer_id_and_product_id(customer_id, product_id)
+    if not cart_item:
+        app.logger.info("Customer id and product id for update have not been found")
+        return jsonify("customer id and product id for update have not been found"), status.HTTP_400_BAD_REQUEST
+
+    requested_quantity = int(request.get_json()["quantity"])
+    # bounds check
+    if requested_quantity < 0:
+        app.lgoger.info('Negative quantity requested')
```
Comment (ytc338) [inline service/service.py]: typo here

Comment (ytc338) [reply service/service.py]: logger

Comment (ytc338) [reply service/service.py]: also I think the minimal quantity should be 1 not 0

Comment (hh1834) [reply service/service.py]: yeah thx what happened was I added bunch of logger statements before pushing. I just changed the quantity too.

**Review by rishabhydv — COMMENTED** [review]
Add a key to jsonify response

**Comment by hh1834** [comment]
got it

**Pull request merged**